internal/restorer/restorer.go

```diff
@@ -253,7 +253,9 @@ func (res *Restorer) RestoreTo(ctx context.Context, dst string) error {
 				idx.Add(node.Inode, node.DeviceID, location)
 			}
 
-			filerestorer.addFile(location, node.Content, int64(node.Size))
+			// Check if target exists and has the right contents
+			matches, _ := res.verifyFile(target, node)
```

Comment:
Unchecked error.

Reply:
This is intentional as matches is always returned. If there is any problem with checking the file, there will be no matches and the file will be fully restored. In fact, here is no error case to check.